src/operations/mutations/resetPassword.ts

```diff
@@ -0,0 +1,45 @@
+import { gql, useMutation } from '@apollo/client';
+import { Dispatch, SetStateAction } from 'react';
+import { useSetRecoilState } from 'recoil';
+
+import { AUTHENTICATION_RESPONSE_FIELDS } from '@src/fragments/user';
+import { toastsState } from '@src/recoils';
+import { Mutation, MutationChangePasswordArgs } from '@src/types/graphql';
+
+export const CHANGE_PASSWORD = gql`
+  ${AUTHENTICATION_RESPONSE_FIELDS}
```

Comment:
`AUTHENTICATION_RESPONSE_FIELDS ` 이 친구는 필요 없어보입니다!
```suggestion
import { toastsState } from '@src/recoils';
import { Mutation, MutationChangePasswordArgs } from '@src/types/graphql';

export const CHANGE_PASSWORD = gql`
```